.github/workflows/bump-version.yml

```
@@ -0,0 +1,33 @@
+name: Versioning
+
+on:
+  pull_request:
+
+jobs:
+  bump-version:
+    name: Check version has been bumped
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout to BASE
+        uses: actions/checkout@v4
+        with:
+          ref: dev
+
+      - name: Get BASE_VERSION
+        run: |
+          echo "BASE_VERSION=$(grep '"version"' apps/web/package.json | sed -E 's/.*"version": *"([^"]+)".*/\1/')" >> $GITHUB_ENV
+
+      - name: Checkout to HEAD
+        uses: actions/checkout@v4
+
+      - name: Get HEAD_VERSION
+        run: |
+          echo "HEAD_VERSION=$(grep '"version"' apps/web/package.json | sed -E 's/.*"version": *"([^"]+)".*/\1/')" >> $GITHUB_ENV
+
+      - name: Compare versions
+        run: |
+          if [[ ! "$HEAD_VERSION" > "$BASE_VERSION" ]];
+          then echo ""$HEAD_VERSION" is not greater than "$BASE_VERSION"" && exit 1;
+          else echo "New version is: "$HEAD_VERSION""
+          fi;
```

```suggestion
          if [[ ! "$HEAD_VERSION" > "$BASE_VERSION" ]]; then
            echo ""$HEAD_VERSION" is not greater than "$BASE_VERSION"" && exit 1;
          else
            echo "New version is: "$HEAD_VERSION""
          fi
```

(just that I'm more used to this indentation)